selection-plugin/tests/counter/counter.golden.txt

```diff
@@ -0,0 +1,3 @@
+1
```

Comment:
This golden looks broken now, what happened?

Reply:
I see the problem.  The golden should probably have actually output from `selection_to_tcl_list` and generate a TCL error if the comparision to `-write` fails

Reply:
Fixed, the reference contains the selection now and the test returns a tcl error upon failure.